internal/controllers/topology/cluster/desired_state_test.go

```diff
@@ -512,8 +530,31 @@ func TestComputeControlPlaneVersion(t *testing.T) {
 		}).
 		Build()
 
+	nonBlockingBeforeClusterUpgradeResponse := &runtimehooksv1.BeforeClusterUpgradeResponse{
+		CommonRetryResponse: runtimehooksv1.CommonRetryResponse{
+			CommonResponse: runtimehooksv1.CommonResponse{
+				Status: runtimehooksv1.ResponseStatusSuccess,
```

Comment:
Let's try to be consistent in the tests. I would suggest to consistently set Status, Message and RetryAfterSeconds only if they don't have the zero value.

(here it's fine, please check the unit tests of the other hooks)